dashboards/kubernetes.node.element.detail.json

```diff
@@ -0,0 +1,139 @@
+        "showBands" : "true",
+        "metricUnit" : "percentunit",
+        "elementScopeAttributes" : "[]",
+        "showHighest" : "true",
+        "categories" : "[]",
+        "tableColumns" : "[{\"columnType\":\"elementType\",\"width\":\"10%\"},{\"columnType\":\"elementName\",\"width\":\"80%\"},{\"columnType\":\"metric\",\"width\":\"10%\",\"metricDisplayName\":null,\"metricFqn\":null,\"metricAggFn\":null,\"metricAgg\":null,\"metricUnit\":null}]",
+        "colorByMetric" : "false",
+        "elementScopeTypes" : "[\"Kubernetes Node\"]",
+        "grouping" : "attribute=SERVICE",
+        "useAllMetricScopeTags" : "true",
+        "metricAggs" : "[]",
+        "elementScopeIds" : "[]",
+        "metricScopeTags" : "[]",
+        "groupByPolicy" : "false",
+        "useAllElementScopeAttributes" : "true",
+        "metricAgg" : "avg",
+        "metrics" : "[{\"fqn\":\"memory.node_utilization\",\"useRegex\":false,\"aggFns\":[\"avg\"],\"aggFn\":null,\"groupAggFn\":null,\"aggregationGroups\":[]}]",
+        "topNLimit" : "5",
+        "elementScopeExcludedTags" : "[]"
+      }
+    }, {
+      "id" : "6bf58951-508a-4b54-bd81-9b360339634f",
+      "name" : "CPU Utilization",
+      "description" : null,
+      "widgetType" : "multi-metric",
+      "properties" : {
+        "visualization" : "line",
+        "showElementTotal" : "true",
+        "elementScopeTags" : "[]",
+        "policies" : "[]",
+        "useAllElementScopeTags" : "true",
+        "metricLimit" : "10",
+        "showBands" : "true",
+        "metricUnit" : "percentunit",
+        "elementScopeAttributes" : "[]",
+        "showHighest" : "true",
+        "categories" : "[]",
+        "tableColumns" : "[{\"columnType\":\"elementType\",\"width\":\"10%\"},{\"columnType\":\"elementName\",\"width\":\"80%\"},{\"columnType\":\"metric\",\"width\":\"10%\",\"metricDisplayName\":null,\"metricFqn\":null,\"metricAggFn\":null,\"metricAgg\":null,\"metricUnit\":null}]",
+        "colorByMetric" : "false",
+        "elementScopeTypes" : "[\"Kubernetes Node\"]",
+        "grouping" : "attribute=SERVICE",
+        "useAllMetricScopeTags" : "true",
+        "metricAggs" : "[]",
+        "elementScopeIds" : "[]",
+        "metricScopeTags" : "[]",
+        "groupByPolicy" : "false",
+        "useAllElementScopeAttributes" : "true",
+        "metricAgg" : "avg",
+        "metrics" : "[{\"fqn\":\"cpu.node_utilization\",\"useRegex\":false,\"aggFns\":[\"avg\"],\"aggFn\":null,\"groupAggFn\":null,\"aggregationGroups\":[]}]",
+        "topNLimit" : "5",
+        "elementScopeExcludedTags" : "[]"
+      }
+    }, {
+      "id" : "5f3534fb-4538-4847-8ff0-229fb4ea809b",
+      "name" : "Tags",
+      "description" : null,
+      "widgetType" : "element-tags",
+      "properties" : {
+        "showElementTotal" : "true",
+        "selectedAttributes" : "[]",
+        "elementScopeTags" : "[]",
+        "useElementNameContains" : "true",
+        "element_id" : "a3915a25-0d84-31e4-81c4-6163bd186952",
```

Comment:
do you need to delete the element id?

Reply:
I will, even though I copied this portion from other packages and it will be ignored by UI since it's element detail dashboard and UI knows which element it is in the url.